feature/sponsors/src/main/java/io/github/droidkaigi/confsched2023/sponsors/section/SponsorList.kt

```diff
@@ -24,11 +31,15 @@ private const val SINGLE_ITEM_SPAN_COUNT = 6
 private const val DOUBLE_ITEM_SPAN_COUNT = 6 / 2
 private const val TRIPLE_ITEM_SPAN_COUNT = 6 / 3
 
+data class SponsorListUiState(
```

Comment:
I created a UiState for SponsorList, but do you think it's redundant?
I created the SponsorList's UiState, thinking it might be necessary since the Section is also managed by the UiState.

Reply:
I was about to create this issue. It was in my issue stock 😄 Thanks!